packages/paste-libraries/react-textarea-autosize/src/useWindowResizeListener.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import useLatest from 'use-latest';
```

Comment:
question: Is this being used to check the value of the ref?

suggestion(conditional): if that is the case, this is a pretty simple hook to implement. I suggest implementing it under our utilities rather than adding a new dependency.

Reply:
Counterpoint, I'd rather the team utilize well tested third parties, than create more code to maintain.

Reply:
I'm going to skip doing this for now, just because its Good Enough™ and I copied it over from the original library.